packages/asc-ui-beta/src/utils/__test__/betaUtils.test.tsx

```diff
@@ -1,7 +1,7 @@
 import type { ElementType } from 'react'
 import { render, screen } from '@testing-library/react'
 import styled from 'styled-components'
-import { ThemeProvider, breakpoint } from '../../../../asc-ui'
+import { ThemeProvider, breakpoint } from '@amsterdam/asc-ui/es'
```

Comment:
(Be careful with this. If this were a component file the path would not resolve after compilation to ES. In a test file it should be ok.)